app/common/data/models.py

```diff
@@ -155,6 +157,32 @@ class CollectionSchema(BaseModel):
     __table_args__ = (UniqueConstraint("name", "grant_id", "version", name="uq_schema_name_version_grant_id"),)
 
 
+class CollectionStatusEnum(enum.StrEnum):
+    NOT_STARTED = "Not started"
+
+
+class Collection(BaseModel):
+    __tablename__ = "collection"
+
+    data: Mapped[json_scalars]
```

Comment:
I can't think of anything else this would be, tiny simplifier for the interface
```suggestion
    data: Mapped[json_scalars] = mapped_column(default={})
```